packages/wallet-core/src/jwt-session/index.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as v from 'valibot';
+import type { JWTPayload } from 'jose';
+import { SignJWT, decodeJwt, jwtVerify } from 'jose';
+
+const AccountSchema = v.object({
+	address: v.string(),
+	publicKey: v.string(),
+	nickname: v.optional(v.string()),
+	icon: v.optional(v.string()),
```

Comment:
what is this? I've never seen accounts with icons

Reply:
Remove this as well